index.js

```diff
@@ -147,9 +204,20 @@ var installedVersion = function(matching) {
 var match = function(version) {
   return Promise.resolve()
   .then(function() { return installedVersion(version); })
-  .then(function(use) {
+  .then(function(useVersion) {
+    return useVersion === 'system' ?
+      getSystemNode()
+        .then(parseMatching)
+        .then(function(use) {
+          return 'system: ' + use;
+        }) :
+      useVersion;
+  })
+  .then(function(useVersion) {
+    var use = (useVersion && useVersion.indexOf('system') === 0) ?
+      'system' : useVersion;
```

Comment:
This has the ability to conflict with an alias defined as something like `system-x`, doesn’t it?

Perhaps the `then` handler above that decides to get the system node should also update a variable define in the scope of the `match` function body to store that choice & use it here.

Reply:
@wbyoung Actually, in this case `useVersion` will only ever start with 'system' if the `.nvmrc` string is simply `system`, and not if it is something like `system-x`. The reason is that `resolveVersion` will resolve to an actual version number for aliases like `system-x`, which will then be propagated throughout the rest of the `match` method. I've added a test to check that this is the case in the last commit.